packages/firebase_dynamic_links/test/firebase_dynamic_links_test.dart

```diff
@@ -0,0 +1,415 @@
+                'iosParameters': null,
+                'itunesConnectAnalyticsParameters': null,
+                'link': 'test-link.com',
+                'navigationInfoParameters': <String, dynamic>{
+                  'forcedRedirectEnabled': true,
+                },
+                'socialMetaTagParameters': null,
+              }),
+          isMethodCall('DynamicLinkParameters#buildShortLink',
+              arguments: <String, dynamic>{
+                'androidParameters': null,
+                'domain': 'test-domain',
+                'dynamicLinkParametersOptions': null,
+                'googleAnalyticsParameters': null,
+                'iosParameters': null,
+                'itunesConnectAnalyticsParameters': null,
+                'link': 'test-link.com',
+                'navigationInfoParameters': <String, dynamic>{
+                  'forcedRedirectEnabled': true,
+                },
+                'socialMetaTagParameters': null,
+              }),
+        ]);
+      });
+
+      test('$SocialMetaTagParameters', () async {
+        final DynamicLinkParameters components = new DynamicLinkParameters(
+            domain: 'test-domain',
+            link: Uri.parse('test-link.com'),
+            socialMetaTagParameters: SocialMetaTagParameters(
+              description: 'describe',
+              imageUrl: Uri.parse('thisimage'),
+              title: 'bro',
+            ));
+
+        await components.buildUrl();
+        await components.buildShortLink();
+
+        expect(log, <Matcher>[
+          isMethodCall('DynamicLinkParameters#buildUrl',
+              arguments: <String, dynamic>{
+                'androidParameters': null,
+                'domain': 'test-domain',
+                'dynamicLinkParametersOptions': null,
+                'googleAnalyticsParameters': null,
+                'iosParameters': null,
+                'itunesConnectAnalyticsParameters': null,
+                'link': 'test-link.com',
+                'navigationInfoParameters': null,
+                'socialMetaTagParameters': <String, dynamic>{
+                  'description': 'describe',
+                  'imageUrl': 'thisimage',
+                  'title': 'bro',
+                },
+              }),
+          isMethodCall('DynamicLinkParameters#buildShortLink',
+              arguments: <String, dynamic>{
+                'androidParameters': null,
+                'domain': 'test-domain',
+                'dynamicLinkParametersOptions': null,
+                'googleAnalyticsParameters': null,
+                'iosParameters': null,
+                'itunesConnectAnalyticsParameters': null,
+                'link': 'test-link.com',
+                'navigationInfoParameters': null,
+                'socialMetaTagParameters': <String, dynamic>{
+                  'description': 'describe',
+                  'imageUrl': 'thisimage',
+                  'title': 'bro',
+                },
+              }),
+        ]);
```

Comment:
A few trailing commas (after `]` I believe) should help reduce all indentation to two spaces in the above code.